### PR TITLE
FIX: electrs-esplora 'verbose transactions are currently unsupported'…

### LINKED ISSUE #3985: Electrum server doesn't support Verbose param in blockchainTransaction_get call
Electrum server like, `electrum.blockstream.info:60002` (testnet) doesn't support VERBOSE param in `blockchainTransaction_get`. It throws `verbose transactions are currently unsupported` error and BW doesn't handle it correctly. Instead it just can't load transactions list and fails silently 

`electrum.blockstream.info:60002` uses `electrs-esplora 0.4.1`, `ElectrumX 1.16.0` works fine

### PR DESCRIPTION
**Pull request by Overtorment**
… (closes #3985)

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2362h6145q8pj0rzqz7epncp44?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9706)

File: blue_modules/BlueElectrum.js [hunk 1]
```diff
@@ -382,11 +382,37 @@ module.exports.getTransactionsFullByAddress = async function (address) {
...
+      full = await mainClient.blockchainTransaction_get(tx.tx_hash, true);
+    } catch (error) {
+      if (String(error?.message ?? error).startsWith('verbose transactions are currently unsupported')) {
+        // apparently, stupid esplora instead of returning txhex when it cant return verbose tx started
+        // throwing a proper exception. lets fetch txhex manually and decode on our end
+        const txhex = await mainClient.blockchainTransaction_get(tx.tx_hash, false);
+        full = txhexToElectrumTransaction(txhex);
+      } else {
+        // nope, its something else
+        throw new Error(String(error?.message ?? error));
```
Comment (limpbrains) [inline blue_modules/BlueElectrum.js:396]: Why reconstruct exception. Just throw existing error

File: blue_modules/BlueElectrum.js [hunk 2]
```diff
@@ -382,11 +382,37 @@ module.exports.getTransactionsFullByAddress = async function (address) {
...
+        prevTxForVin = await mainClient.blockchainTransaction_get(input.txid, true);
+      } catch (error) {
+        if (String(error?.message ?? error).startsWith('verbose transactions are currently unsupported')) {
+          // apparently, stupid esplora instead of returning txhex when it cant return verbose tx started
+          // throwing a proper exception. lets fetch txhex manually and decode on our end
+          const txhex = await mainClient.blockchainTransaction_get(input.txid, false);
+          prevTxForVin = txhexToElectrumTransaction(txhex);
+        } else {
+          // nope, its something else
+          throw new Error(String(error?.message ?? error));
```
Comment (limpbrains) [inline blue_modules/BlueElectrum.js:413]: Same

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: blue_modules/BlueElectrum.js (hunk 1 above)
Comment (Overtorment) [reply blue_modules/BlueElectrum.js:396]: i noticed that it is not a proper error object, for some reason its just a string (thus, this `error?.message ?? error` construct).

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**